test/JDBC/expected/babel_isnumeric-vu-verify.out

```diff
@@ -229,7 +229,7 @@ select isnumeric(' ')
 GO
 ~~START~~
 int
-0
+1
```

Comment:
Is this expected output?

Reply:
yes, this is expected as per TSQL doc, empty string or white spaces can be converted to money datatype.